test/token/ERC721/ERC721.behavior.js

```diff
@@ -9,901 +9,837 @@ const { RevertType } = require('../../helpers/enums');
 const firstTokenId = 5042n;
 const secondTokenId = 79217n;
 const nonExistentTokenId = 13n;
-const fourthTokenId = 4n;
-const baseURI = 'https://api.example.com/v1/';
 
 const RECEIVER_MAGIC_VALUE = '0x150b7a02';
 
-function shouldBehaveLikeERC721() {
-  beforeEach(async function () {
-    const [owner, newOwner, approved, operator, other] = this.accounts;
-    Object.assign(this, { owner, newOwner, approved, operator, other });
-  });
-
-  shouldSupportInterfaces(['ERC721']);
-
-  describe('with minted tokens', function () {
+function shouldBehaveLikeERC721(extraTxTests) {
+  describe('ERC712 behavior', function () {
```

Comment:
if we want to add that (I'm not sure of that), then the name should be something like "behaves like an ERC-721" or "follows the ERC-721 standard".